charts/itsm-ng/values.yaml

```
@@ -86,17 +86,24 @@ tolerations: []
 affinity: {}
 
 
-itsm:
-  TZ: Europe/Paris
+TZ: Europe/Paris
+
+auth:
+  adminPassword: ""
   database:
     name: itsm
     user: itsm
     password: ""
-  adminPassword: ""
+
+persistence:
+  enabled: false
```

PVC should be enabled as a default, because it would probably not work as expected otherwise.
```suggestion
  enabled: true
```